events/quarkus/build.gradle.kts

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (C) 2022 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+  `java-library`
+  jacoco
+  `maven-publish`
+  signing
+  alias(libs.plugins.quarkus)
+  `nessie-conventions`
+}
+
+extra["maven.name"] = "Nessie - Events - Quarkus"
+
+dependencies {
+  implementation(project(":nessie-versioned-spi"))
+  implementation(project(":nessie-events-api"))
+  implementation(project(":nessie-events-spi"))
+  implementation(project(":nessie-events-service"))
+
+  // Quarkus
+  implementation(enforcedPlatform(libs.quarkus.bom))
+  implementation("io.quarkus:quarkus-vertx")
+
+  // Metrics
+  implementation(libs.micrometer.core)
+
+  // OpenTelemetry
+  implementation(platform(libs.opentelemetry.bom))
```

Comment:
I think, the Quarkus-platform-BOM already defines the necessary versions (can remove the otel bom dependency)